include/rc_client.h

```diff
@@ -317,6 +317,91 @@ typedef struct rc_client_subset_t {
 
 RC_EXPORT const rc_client_subset_t* RC_CCONV rc_client_get_subset_info(rc_client_t* client, uint32_t subset_id);
 
+/*****************************************************************************\
+| Fetch Game Hashes                                                           |
+\*****************************************************************************/
+
+typedef struct rc_client_hash_library_entry_t
+{
+  /* The hash for the game */
```

Comment:
Per field comments are not provided in this file. The names should be descriptive enough to not require a comment (if additional detail is needed, it can be included in the wiki). And it allows for delineating version differences.
```
  uint8_t state;
  uint8_t category;
  uint8_t bucket;
  uint8_t unlocked;
  /* minimum version: 11.1 */
  float rarity;
  float rarity_hardcore;
  uint8_t type;
  /* minimum version: 12.0 */
  const char* badge_url;
  const char* badge_locked_url;
```